src/qt/walletcontroller.cpp

```diff
@@ -62,6 +62,20 @@ WalletModel* WalletController::openWallet(const std::string& name, QWidget* pare
     return wallet_model;
 }
 
+void WalletController::closeWallet(WalletModel* wallet_model, QWidget* parent)
+{
+    QMessageBox::StandardButton button = QMessageBox::question(parent, tr("Close wallet"),
```

Comment:
`QMessageBox::question()` is [obsolete](https://doc.qt.io/qt-5.9/qmessagebox-obsolete.html#question-1). Could it be avoided?